migrations/006.do.daily-node-metrics.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE daily_node_metrics (
```

Comment:
Continuing the discussion started in https://github.com/filecoin-station/spark-stats/pull/71#discussion_r1581157857

> Makes sense! Should we also rename the table and references to it in spark-evaluate (it's currently daily_node_metrics)

That's a great idea, let's do it!

Please rename the migration SQL file too.

Other tables are called simply `daily_{things}`, maybe we can do the same here and use the name `daily_stations`?